src/components/Table/__stories__/Table.stories.tsx

```diff
@@ -131,6 +132,27 @@ const WithTableActionsTemplate: StoryFn<TableProps<DataItem>> = (args) => {
 };
 export const HOCWithTableActions = WithTableActionsTemplate.bind({});
 
+// ---------------------------------
+const WithTableActionsRenderRowActionsTemplate: StoryFn<TableProps<DataItem>> = (args) => {
```

Comment:
but it is impossible to combine `getRowActions` and `renderRowActions` at the same time. 
`renderRowActions` will override `getRowActions`

Reply:
No need to! It will be two stories with different props, just like in https://github.com/gravity-ui/uikit/blob/main/src/components/Table/__stories__/Adaptive.tsx .
It will be nice if you render labels for each (in the example above there are no labels and it looks a little messy).

Reply:
[93eb9f6](https://github.com/gravity-ui/uikit/pull/1353/commits/93eb9f6f092fe1f277f4648a3acb6a5105486bb1)